src/components/skeletonUI.js

```diff
@@ -1,12 +1,9 @@
 import React from 'react'
-import loader from '../../assets/loading.gif'
+import loader from './img/loading.gif'
 export default function skeletonUI() {
   return (
     <div className="flex flex-col items-center">
-      <img src={loader} alt='' />
+      <img src={loader} alt='' className='pt-32 w-72 h-70'/>
```

Comment:
remove this className ,your loader will look good

Reply:
Good